activation/poet.go

```diff
@@ -480,7 +485,7 @@ func (c *poetService) Submit(
 
 	logger.Debug("submitting challenge to poet proving service")
 
-	submitCtx, cancel := withConditionalTimeout(ctx, c.requestTimeout)
+	submitCtx, cancel := withConditionalTimeout(ctx, c.submitChallengeRequestTimeout)
```

Comment:
We want to retry to submit until the poet round starts (a point in time, a deadline), not for a certain duration. We might start submitting even 2 minutes before the poet round starts, depending on the duration of PoST proving. It makes no sense to retry after the round started.

Also, it should retry infinitely, or up to the configured retry limit (different retry limit than for getting the proof).

Reply:
I completely removed this timeout as we discussed